ida-chatbot/src/app/app.component.ts

```diff
@@ -18,11 +18,12 @@ import {IdaEventService} from './service/event/ida-event.service';
   styleUrls: ['./app.component.css']
 })
 export class AppComponent {
-  idCount = 1;
+  idCount = 2;
   title = 'app';
   public introSideItem = new SidebarElement(0, 'Introduction', 'intro');
+  public ontologySideItem = new SidebarElement(1, 'Ontology Explorer', 'ontology');
   public activeItem = 0;
-  private sidebarItems: SidebarElement[] = [this.introSideItem];
+  private sidebarItems: SidebarElement[] = [this.introSideItem, this.ontologySideItem];
```

Comment:
Good work 👍 is it hardcoded? or it will add dynamically? and the thing is AFAIR we don't need any sidebar tab or any tab for RDF ontology explorer as we just only need to show it under the table explorer?

Reply:
And also there is a conflict :)

Reply:
Hi Maqbool,

for hard coded things @nandeeshpatelgp  is best person to answer it, and  for where to display it, for now me and Himadri have decided to have a tab below the Introduction and on click of it load the ontology graph component and it will display as 
![image](https://user-images.githubusercontent.com/17275639/60592695-77fee600-9da1-11e9-9990-17d20cb48732.png)

we know that this is not the right place to display the ontology graph but this cannot be implement at the last minute , I have created an issue "https://github.com/dice-group/ida/issues/165" which includes the pending tasks for this PR, please add this into it by mentioning how and where it should be displayed.

And please mention more details on what conflict is there :).

Thanks and regards,
Rahul sherikar.

Reply:
Thanks ill wait for @nandeeshpatelgp comment :)

Reply:
Hi @maqboolkhan 
1. For now I think it is a independent component and not related to display table as the data used for ontology is historians data which we are not using on the application yet. This change is just for demo.
2. I don't understand exactly when you say hard-coded, I have made the ontology explorer as a sidebar element similar to intro, which will be shown when you click on the ontology explorer option on sidebar and the option on sidebar is shown all the time similar to introduction. Basically sidebar will have 2 options (intro, ontology) all the time and the third entry will be dynamic for dataset which will be added when user loads any dataset using the chatbot.

I hope this answers your question. If not please let me know.